### ISSUE TITLE
Upgrade dependencies post Grommet release v2.23.0

### ISSUE THREAD

**Issue by britt6612** [grommet/hpe-design-system#2552]

**Comment by jcfilben** [comment]
Jest upgrade is blocked by ts-jest since it doesn't support the latest version of Jest yet. Created a new issue to track this: https://github.com/grommet/hpe-design-system/issues/2556

**Comment by jcfilben** [comment]
Dependencies have been upgraded besides Jest, which I created a separate issue for
